upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.12.yaml.template

```diff
@@ -137,6 +137,7 @@ data:
   auto-direct-node-routes: "{{- if .AutoDirectNodeRoutes -}}true{{- else -}}false{{- end -}}"
   enable-node-port: "{{- if .EnableNodePort -}}true{{- else -}}false{{- end -}}"
   kube-proxy-replacement: "{{- if .EnableNodePort -}}strict{{- else -}}partial{{- end -}}"
+  enable-remote-node-identity: "{{- if .EnableRemoteNodeIdentity -}}true{{- else -}}false{{- end -}}"
```

Comment:
is it okay that this field is present even if running cilium <1.7.0 ?

Reply:
Yeah. Cilium just ignores it.